pkg/tenant/auth.go

```diff
@@ -139,13 +142,53 @@ func (s *Server) Login(c *gin.Context) {
 		Password: params.Password,
 	}
 
+	ok := hasInviteToken(params.InviteToken)
+
+	if ok {
+		req.InviteToken = params.InviteToken
+	}
+
 	var reply *qd.LoginReply
 	if reply, err = s.quarterdeck.Login(c.Request.Context(), req); err != nil {
 		sentry.Debug(c).Err(err).Msg("tracing quarterdeck error in tenant")
 		api.ReplyQuarterdeckError(c, err)
 		return
 	}
 
+	// Update the status of a member with an invite token to Confirmed.
+	if ok && reply.AccessToken != "" {
+		// Parse access token to get the orgID.
+		var claims *jwt.RegisteredClaims
+		if claims, err = tokens.ParseUnverified(reply.AccessToken); err != nil {
```

Comment:
So looks like we should create another method in the `tokens` package which passes in the token claims instead of the jwt claims:

```golang
func ParseUnverifiedTokenClaims(tks string) (claims *tokens.Claims, err error) {
    claims := &tokens.Claims{}
    if _, _, err = tsparser.ParseUnverified(tks, claims); err != nil {
        return nil, err
    }
    return claims, nil
}
```